src/scripts/add-user-to-group.test.ts

```diff
@@ -43,7 +43,7 @@ describe('Adding a user to a group', () => {
   });
 
   it('fails if the user does not exist', async () => {
-    // Given a user service that can not locate the user.
+    // Given a user service that cannot locate the user.
```

Comment:
**future:** don't make these sorts of stylistic changes inside another PR. It makes them slower to review - and reducing review cycle time is a key goal for us.

Reply:
ok i will keep this in mind for future